### PR TITLE
Chore: update deps

### PR DESCRIPTION
**Pull request by joshcanhelp**
Update dependencies and fix issues that cropped up.

### CODE REVIEW

File: src/rules/mandrill.js [hunk 1]
```diff
@@ -42,22 +42,21 @@ function sendMandrillEmail(user, context, callback) {
   const mandrill_send_endpoint =
     'https://mandrillapp.com/api/1.0/messages/send.json';
 
-  request.post({ url: mandrill_send_endpoint, form: body }, function (
```
Comment (joshcanhelp) [inline src/rules/mandrill.js:45]: Formatting

File: src/rules/linkedin-original-picture.js [hunk 2]
```diff
@@ -19,8 +19,7 @@ function useOriginallinkedInProfilePicture(user, context, callback) {
   const liIdentity = _.find(user.identities, { connection: 'linkedin' });
 
   const options = {
-    url:
```
Comment (joshcanhelp) [inline src/rules/linkedin-original-picture.js:22]: Formatting

File: src/rules/incognia-authentication.js [hunk 3]
```diff
@@ -17,11 +17,8 @@ async function incogniaAuthenticationRule(user, context, callback) {
 
   const { IncogniaAPI } = require('@incognia/api@1.1.1');
 
-  const {
```
Comment (joshcanhelp) [inline src/rules/incognia-authentication.js:20]: Formatting

File: src/rules/mongodb.js [hunk 4]
```diff
@@ -11,21 +11,22 @@ function enrichProfileWithMongo(user, context, callback) {
   const MongoClient = require('mongodb@3.1.4').MongoClient;
   const mongoUrl = configuration.MONGO_CONNECTION_STRING;
 
-  MongoClient.connect(mongoUrl, { useNewUrlParser: true }, function (
```
Comment (joshcanhelp) [inline src/rules/mongodb.js:14]: Formatting

File: src/rules/parse.js [hunk 5]
```diff
@@ -42,9 +42,8 @@ function generateParseSessionToken(user, context, callback) {
 
       // user was found, add sessionToken to user profile
       if (response.statusCode === 200) {
-        context.idToken['https://example.com/parse_session_token'] = JSON.parse(
```
Comment (joshcanhelp) [inline src/rules/parse.js:45]: Formatting

File: src/rules/soap-webservice.js [hunk 6]
```diff
@@ -17,8 +17,7 @@ function getRolesFromSoapService(user, context, callback) {
     request.post(
       {
         url: 'https://somedomain.com/RoleService.svc',
-        body:
```
Comment (joshcanhelp) [inline src/rules/soap-webservice.js:20]: Formatting

File: .circleci/config.yml [hunk 7]
```diff
@@ -5,7 +5,7 @@ jobs:
     parameters:
       v:
         type: string
-        default: "8"
+        default: "12"
```
Comment (joshcanhelp) [inline .circleci/config.yml:8]: This is now checking Node v12 twice. If I remove the duplicate job then CI just hangs.

I tried to add Node 16 for fun but get a message about not having a password ...

**Review by jpadilla — APPROVED** [review]

**Pull request merged**